user/howto/launch-dask-gateway-cluster.md

```diff
@@ -0,0 +1,225 @@
+  <source src="../../../_static/videos/dask-compute-kmeans.mp4" type="video/mp4" />
+  <p>Video showing dask dashboard while computing k-means clustering.</p>
+</video>
+
+<br>
+
+Plot the results.
+
+```python
+fig, ax = plt.subplots()
+ax.scatter(X[::10000, 0], X[::10000, 1], marker='.', c=km.labels_[::10000],
+           cmap='viridis', alpha=0.25);
+```
+
+:::{image} media/dask-plot-result.png
+:alt: Plot of the k-means clustering results.
+:align: center
+:width: 75%
+:::
+
+### Shut down the cluster
+
+Shut down the cluster when in not in use to minimize the waste of computational resources and costs.
+
+```python
+cluster.close()
+```
+
+## FAQs
+
+- *What are the best practices for using Dask to scale computations?*
+
+  Take a look at the [Dask - Best Practices](https://docs.dask.org/en/latest/best-practices.html) documentation for a guide to using Dask efficiently.
+
+- *Why can't I choose the instance type for my cluster?*
+
+  The default is `n2-high-mem16` for Google Cloud and `r5.4xlarge` for AWS, with a maximum of 16 CPUs available on each machine. These instance types are chosen so that there is a consistent node pool across all 2i2c Dask hubs and more efficient cluster resourcing.
+
+- *Should I use manual or adaptive scaling?*
+
+  We suggest starting with adaptive scaling to dynamically scale up the number of workers when necessary but scale down and save resources when not actively computing. Once you are confident about your code's performance, you can use the manual scaling for fine-grained control over the number of workers needed.
+
+- *How do I choose the number of workers/resources per worker?*
+
+  Like any other scaling problem, the answer to this question is specific to the application code. As suggested in the [Dask - Best Practices](https://docs.dask.org/en/latest/best-practices.html), start with a small subset of your data and evaluate performance using the Dask dashboard before scaling resources to the full dataset.
+
+- *Why are there limited options for CPU and memory resources per worker?*
+
+  The limited options of 1/2/4/8/16 CPUs and associated memory requests are chosen to enable the most efficient use of the node pool available. Otherwise, for example, a user could accidentally request 51% of a machine's resources per worker so only 1 worker can fit on an entire machine. In this example, if 2 workers were needed for the computation then this increases cloud costs for the 2 machines required while potentially leaving 49% of resources idle per machine.
+  
+- *Why did my kernel die?*
+
+  There can be many reasons for this, but the most common one when it comes to Dask is an out-of-memory error caused by committing data into memory that exceeds the available RAM limit. Try using Dask to scale computations on smaller datasets and write the intermediate results to disk in the {ref}`/tmp folder<filesystem:tmp>`.
+
+- *Which image should I use for the software environment?*
```

Comment:
The user-server image needs `dask-gateway` installed for the dask-gateway client, and `distributed` for the client to interact with the dask-scheduler.

`dask-labextension` provides the dashboarding stuff on the left panel in jupyterlab, so that is also nice.

Reply:
Okay! I will expand on this in the suggestion below.

Reply:
Resolved in https://github.com/2i2c-org/docs/pull/228/commits/45fb24a55c622dd3ac00307fe4ccfc41cb258c99